plugin/adsk/scripts/AEmayaUsdProxyShapeBaseTemplate.mel

```diff
@@ -92,7 +122,8 @@ global proc AEmayaUsdProxyShapeBaseTemplate( string $nodeName )
             // pass it in order to get the current nodename. Cannot use the input one here as it
             // changes based on which proxy shape the AE is viewing.
             editorTemplate -callCustom "AEmayaUsdProxyShapeInfoNew" "AEmayaUsdProxyShapeInfoReplace" inStageData;
-            //editorTemplate -addControl "loadPayloads";
+
+            editorTemplate -callCustom "AEmayaUsdProxyShapePayloadsNew" "AEmayaUsdProxyShapePayloadsReplace" loadPayloads;
```

Comment:
Replace the loadPayloads control (checkbox) with a custom control containing the label/buttons.